### PR TITLE
ADEN-5002 Adds global for enabling FAN

### PR DESCRIPTION
**Pull request by leshniak**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/spec/lookup/prebid/adaptersRegistry.spec.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+/*global describe, it, modules, expect, spyOn, beforeEach*/
+/*jshint maxlen:200*/
+describe('adaptersRegistry', function () {
+	'use strict';
+
+    function AdapterMock(name, enabled) {
+        this.getName = function getName() {
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/spec/lookup/prebid/adaptersRegistry.spec.js]: MAJOR Make this function anonymous by removing its name: 'function() {...}'. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ANamedFunctionExpression)

File: extensions/wikia/AdEngine/js/spec/lookup/prebid/adaptersRegistry.spec.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            return name;
+        };
+
+        this.isEnabled = function isEnabled() {
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/spec/lookup/prebid/adaptersRegistry.spec.js]: MAJOR Make this function anonymous by removing its name: 'function() {...}'. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ANamedFunctionExpression)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/AdEngine/AdEngine2ContextService.class.php [hunk 3]
```diff
@@ -79,6 +79,7 @@ public function getContext( Title $title, $skinName ) {
 				] ),
 				'providers' => $this->filterOutEmptyItems( [
 					'evolve2' => $wg->AdDriverUseEvolve2,
+					'audienceNetwork' => $wg->AdDriverUseAudienceNetworkBidder,
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/AdEngine2ContextService.class.php:82]: What does happen when `$AdDriverUseAudienceNetworkBidder` is not set?

File: maintenance/wikia/setWikiFactoryVariable.php [hunk 4]
```diff
@@ -82,7 +82,7 @@ public function execute() {
 		}
 
 		if ( !empty( $wikiId ) ) {
-			$wikiIds = [ $wikiId ];
+			$wikiIds = explode(',', $wikiId);
```
Comment (nandy-andy) [inline maintenance/wikia/setWikiFactoryVariable.php:85]: Nice :) I thought initially we'll need more changes in the maintenance script :+1:

File: extensions/wikia/AdEngine/AdEngine2ContextService.class.php (hunk 3 above)
Comment (nandy-andy) [reply extensions/wikia/AdEngine/AdEngine2ContextService.class.php:82]: Oh, wait. It's being set by default to `false` in the backend, so I think there is no need to panic ;) https://github.com/Wikia/app/pull/13291/files#diff-013f1f772182745f48c5f1429d0cdd35R1254

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**